e2e/cypress/tests/integration/enterprise/cloud/billing/cloud_pricing_modal_spec.js

```diff
@@ -227,27 +219,27 @@ describe('Pricing modal', () => {
         // # Open the pricing modal
         cy.get('#UpgradeButton').should('exist').click();
 
-        // *Pricing modal should be open
+        // * Pricing modal should be open
         cy.get('#pricingModal').should('exist');
-        cy.get('#pricingModal').get('.PricingModal__header').contains('Select a plan');
+        cy.get('#pricingModal').find('.PricingModal__header').contains('Select a plan');
 
-        // *Check that starter card Downgrade button is disabled
+        // * Check that starter card Downgrade button is disabled
         cy.get('#pricingModal').get('#starter').get('#starter_action').should('be.disabled').contains('Downgrade');
 
-        // *Check that professsional card Upgrade button opens purchase modal
+        // * Check that professsional card Upgrade button opens purchase modal
         cy.get('#pricingModal').get('#professional').get('#professional_action').click();
-        cy.get('.PurchaseModal').should('exist');
+        cy.get('.modal-body').should('exist');
```

Comment:
This class name is used by all modals in Mattermost - you may want to use something a little more specific for the longevity of the test.

Reply:
ohh!! Thank you @nickmisasi ,
I'll test with another locator